daffodil-core/src/main/scala/edu/illinois/ncsa/daffodil/dsom/RealTermMixin.scala

```diff
@@ -70,10 +70,10 @@ trait PropertyReferencedElementInfosMixin {
 }
 
 /**
- * A RealTerm is an element or a sequence or a choice. Group references
- * are excluded as they go away and really have no realization.
+ * A RealTerm is a LocalElementDecl or ElementRef, or
+ * a Sequence or SequenceGroupRef
+ * or a Choice or ChoiceGroupRef
  *
- * Things that apply to real terms, but not group references, go in this trait.
  */
 trait RealTermMixin { self: Term =>
```

Comment:
RealTerm can be collapsed into Term now. All Term are RealTerm, and nothing else is, because ElemetnRef and GroupRef are both Terms now.